modules/assim.sequential/R/downscale_function_hrly.R

```diff
@@ -0,0 +1,96 @@
+#' @title North America Downscale Function
+#' @name NA_downscale_hrly
```

Comment:
Note that in PR #3321 we change the name of the original downscale function to SDA_downscale because there's nothing North America specific about this code.

Reply:
Chiming in with a Roxygen style point: I recommend not including a `@name` line at all -- if you omit it, Roxygen will default to using the function name, which is usually what you want and when they differ it can be confusing. 

Relatedly I recommend leaving the `@title`, `@description`, and `@details` tags implicit: Write the introductory block with no tags, then Roxygen will [take these](https://roxygen2.r-lib.org/articles/rd.html#the-introduction) from the first, second, and third-and-later paragraphs respectively.  In most cases typing the tags out doesn't hurt anything, but can be confusing when they're specified in a different order. We have some legacy functions where I keep looking for the `@details` and being surprised to find them below the parameter descriptions.

None of this changes Mike's point about changing the name of the function itself, just saying it could be changed in one line of this file instead of two.